components/Banner/Banner.styled.ts

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+type MoneyProps = {
+  right?: boolean;
+};
+
+export const Background = styled.section`
+  margin-top: 64px;
+  margin-bottom: -73px;
```

Comment:
I know adding a negative margin is frowned upon, but I wasn't sure how to remove the spacing the page adds to compensate for the navbar when the banner is above the page without performing another path check in the pageLayout.

Reply:
You can make the banner fixed right underneath the navbar as well so that it won't scroll up when you scroll down.